classes/machine.rb

```diff
@@ -0,0 +1,15 @@
+# The machine class provides the information on the actual ATM
+class Machine
+  attr_accessor :balance
+
+  # Verify that the ATM contains sufficient money for the withdrawal
+  def can_withdraw?(amount)
+    amount.class == Fixnum && @balance >= amount && amount > 0
```

Comment:
I would put `amount > 0` before checking the balance (which might be a costly procedure). More in depth answer here:
https://github.com/easmith2/2.3_testing/pull/1#discussion_r33641217